hammer/sim/vcs/__init__.py

```diff
@@ -259,7 +263,9 @@ def run_simulation(self) -> bool:
         if self.level == FlowLevel.RTL and saif_mode != "none":
             find_regs_run_tcl = []
             if saif_mode != "none":
+                stime: Optional[TimeValue] = None
                 if saif_mode == "time":
+                    assert saif_start_time
                     stime = TimeValue(saif_start_time[0])
```

Comment:
There is something fishy here - `saif_start_time` is a string and we're just picking out the first character assuming that the string looks like "3ns"

Reply:
Line 269 seems like a typo. It shouldn't have `[0]` in it (see line 282 - `TimeValue` can be created directly from the string with units).

Reply:
Yes I think you're right, I removed the `[0]`